core/src/test/scala/org/broadinstitute/dsde/rawls/workspace/AggregatedWorkspaceServiceSpec.scala

```diff
@@ -102,14 +104,19 @@ class AggregatedWorkspaceServiceSpec extends AnyFlatSpec with MockitoTestUtils {
   it should "combine WSM data with Rawls data for GCP MC workspaces" in {
     val wsmDao = mock[WorkspaceManagerDAO]
     when(wsmDao.getWorkspace(any[UUID], any[RawlsRequestContext]))
-      .thenReturn(new WorkspaceDescription().gcpContext(new GcpContext().projectId("project-id")))
+      .thenReturn(
+        new WorkspaceDescription()
+          .stage(WorkspaceStageModel.MC_WORKSPACE)
+          .gcpContext(new GcpContext().projectId("project-id"))
+      )
     val svc = new AggregatedWorkspaceService(wsmDao)
 
-    val aggregatedWorkspace = svc.fetchAggregatedWorkspace(mcWorkspace, defaultRequestContext)
+    val aggregatedWorkspace = svc.fetchAggregatedWorkspace(mcRawlsWorkspace, defaultRequestContext)
 
-    aggregatedWorkspace.baseWorkspace shouldBe mcWorkspace
-    aggregatedWorkspace.googleProjectId shouldBe Some(GoogleProjectId("project-id"))
+    aggregatedWorkspace.baseWorkspace shouldBe mcRawlsWorkspace
     aggregatedWorkspace.getCloudPlatform shouldBe WorkspaceCloudPlatform.Gcp
+    aggregatedWorkspace.azureCloudContext shouldBe None
```

Comment:
Reordered asserts throughout to confirm which cloudPlatform first (which IMO is the most important detail), then azure details, then GCP details.